include/net/lwm2m.h

```diff
@@ -544,29 +544,33 @@ struct lwm2m_objlnk {
  * LwM2M clients use this function to modify the pmin attribute
  * for an observation being made.
  * Example to update the pmin of a temperature sensor value being observed:
- * lwm2m_engine_update_observer_min_period("3303/0/5700",5);
+ * lwm2m_engine_update_observer_min_period("3303/0/5700", client_ctx, 5);
  *
  * @param[in] pathstr LwM2M path string "obj/obj-inst/res"
+ * @param[in] client_ctx LwM2M context
  * @param[in] period_s Value of pmin to be given (in seconds).
  *
  * @return 0 for success or negative in case of error.
  */
-int lwm2m_engine_update_observer_min_period(const char *pathstr, uint32_t period_s);
+int lwm2m_engine_update_observer_min_period(const char *pathstr, struct lwm2m_ctx *client_ctx,
```

Comment:
Every function, which takes `struct lwm2m_ctx *` has it as a first parameter. I think we should follow that.